src/rules/consistent-vitest-vi.ts

```diff
@@ -0,0 +1,115 @@
+import { AST_NODE_TYPES, TSESTree } from '@typescript-eslint/utils'
+import { createEslintRule } from '../utils'
+import { HelperName } from '../utils/types'
+import { parseVitestFnCall } from '../utils/parse-vitest-fn-call'
+
+export const RULE_NAME = 'consistent-vitest-vi'
+export type MESSAGE_ID = 'consistentHelper'
+
+const getOppositeHelperKeyword = (helper: HelperName) =>
+  helper === HelperName.vi ? HelperName.vitest : HelperName.vi
```

Comment:
let's rename this function name to something more specific to this rules

Reply:
also can we add this rule to the Readme.md?

Reply:
Please have a look. Tried to change according your suggestions and saw vitest names them `utils`, not `helpers` - so I fixed naming accordingly: https://vitest.dev/api/vi.html